### ISSUE TITLE
`I` undefined in PO's

### ISSUE THREAD

**Issue by stychu** [Codeception/CodeceptJS#1624]
CodeceptJS v2.1.0

```
const { I } = inject();

module.exports = {
  tables: {
    ....
  },
  testCookie() {
    console.log(I)
  }
};
```

I get `undefined` for the I injected into the PO's.  🚨

**Comment by PeterNgTr** [comment]
use old style would work
```
const I = actor();
```

**Comment by stychu** [comment]
How this `const { I } = inject();` works in test file but does not in PO's file

**Comment by PeterNgTr** [comment]
@DavertMik could you give some insights?

**Comment by stychu** [comment]
I even created a new clean project following docs and having the same exact problem. PO's have undefined I from inject()

**Comment by stychu** [comment]
Any help on this? This is very crucial to me right now. If I can't use the inject in the Po's how can I use custom_steps methods then in PO's. `const I = actor();` comes with only default methods and don't have access to the custom_steps file methods

**Comment by stychu** [comment]
Also when I try in one of the PO's doing
`const I = actor(require('../steps_file')());`

Then I have the I with all the default methods and methods from `steps_file` BUT -> when I will do the same with second PO (require the I actor the same way as in the first PO) then I am unable to generate def and run codecept.

```
./cc def
.../acceptance/node_modules/codeceptjs/lib/container.js:263
    throw new Error(`Could not include object ${supportObjectName} from module '${modulePath}'\n${err.message}`);
    ^

Error: Could not include object I from module .../acceptance/steps_file.js'
require(...) is not a function
    at loadSupportObject ...acceptance/node_modules/codeceptjs/lib/container.js:263:11)
    at getSupportObject .../acceptance/node_modules/codeceptjs/lib/container.js:237:12)
    at createSupportObjects ...acceptance/node_modules/codeceptjs/lib/container.js:166:21)
    at Function.create .../acceptance/node_modules/codeceptjs/lib/container.js:35:25)
    at Codecept.init ...acceptance/node_modules/codeceptjs/lib/codecept.js:55:15)
    at Command.module.exports.../acceptance/node_modules/codeceptjs/lib/command/definitions.js:163:12)
    at Command.listener .../acceptance/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:189:13)
    at Command.parseArgs .../acceptance/node_modules/commander/index.js:651:12)
    at Command.parse .../acceptance/node_modules/commander/index.js:474:21)

```

```
./cc run --grep @raiseDeal --plugins wdio --steps

Could not include object I from module '.../acceptance/steps_file.js'
require(...) is not a function

```


Second thing, when I check `inject()` inside the test scenario file it has all the objects included in my codecept.conf.js 

```
include: {
    I: './steps_file.js',
    dashboardPage: './pages/dashboard.js',
    raiseDealPage: './pages/raiseDeal.js',
    modalFragment: './fragments/modal',
    apiConnectorStep: './steps/apiConnector.js',
  },
```

So all methods and objects are here
BUT -> when I do check the `inject)` inside the Page Object Page Fragment Steps Object etc. in all cases the `inject()` is just empty object `{}`

**Comment by stychu** [comment]
@DavertMik  any ideas? I really suffer from this right now

**Comment by PeterNgTr** [comment]
@stychu try this, it should work
```
const I  = actor();

module.exports = {

  // insert your locators and methods here
  google() {
    I.amOnPage('https://google.com');
    I.see('Google');
  }
}
```

```
sysadmins-MacBook-Pro-3:test-codeceptjs thanhnguyen$ ./node_modules/.bin/codeceptjs run --verbose
CodeceptJS v2.1.0
Using test root "/Users/thanhnguyen/Desktop/test-codeceptjs"
Helpers: WebDriver
Plugins: screenshotOnFail

TEST --
    [1] Starting recording promises
    Emitted | suite.before ([object Object])
  test something
    Emitted | test.before ([object Object])
    Emitted | test.start ([object Object])
    Emitted | step.before (I am on page "https://github.com")
    Emitted | step.after (I am on page "https://github.com")
    Emitted | step.before (I see "GitHub")
    Emitted | step.after (I see "GitHub")
    Emitted | step.before (I am on page "https://google.com")
    Emitted | step.after (I am on page "https://google.com")
    Emitted | step.before (I see "Google")
    Emitted | step.after (I see "Google")
    Emitted | step.start (I am on page "https://github.com")
    I: canVisitGithub 
      I am on page "https://github.com"
    Emitted | step.passed (I am on page "https://github.com")
    Emitted | step.finish (I am on page "https://github.com")
    Emitted | step.start (I see "GitHub")
      I see "GitHub"
    Emitted | step.passed (I see "GitHub")
    Emitted | step.finish (I see "GitHub")
    Emitted | step.start (I am on page "https://google.com")
    testPOPage: google 
      I am on page "https://google.com"
    Emitted | step.passed (I am on page "https://google.com")
    Emitted | step.finish (I am on page "https://google.com")
    Emitted | step.start (I see "Google")
      I see "Google"
    Emitted | step.passed (I see "Google")
    Emitted | step.finish (I see "Google")
    Emitted | test.passed ([object Object])
    Emitted | test.finish ([object Object])
  ✔ OK in 1690ms

    Emitted | test.after ([object Object])
  S test something 2
    Emitted | suite.after ([object Object])

  OK  | 1 passed, 1 skipped   // 3s
    Emitted | global.result ([object Object])
    Emitted | global.after ([object Object])
```

**Comment by stychu** [comment]
@PeterNgTr Thanks for suggestions. I managed to found the problem.
When I was trying to Include the steps_file in the actor() I didn't pay attention that I earlier included my PO's in the steps_file when was testing some approaches.
```
//steps_file.js

const { dashboardPage, raiseDealPage } = require('./pages');
const { modalFragment } = require('./fragments');
```

After removing those requires from steps_file I was able to use

`const I = actor(require('../steps_file')());` in all of my Page Objects etc.


The documentation should be fixed or the codebase itself because following the official docs it tells to use `const { I, otherPage } = inject();` also codeceptjs gpo generates those which are not working because `inject()` is an empty object here
